ZAPD/ZRoom/Commands/SetRoomList.cpp

```diff
@@ -105,7 +105,7 @@ std::string RomFile::GetBodySourceCode() const
 					declaration += "\n";
 
 				declaration +=
-					StringHelper::Sprintf("\t{ (u32)_%sSegmentRomStart, (u32)_%sSegmentRomEnd },",
+					StringHelper::Sprintf("\t{ (uintptr_t)_%sSegmentRomStart, (uintptr_t)_%sSegmentRomEnd },",
```

Comment:
I did some small tests in OoT and MM, and this cast doesn't seem necessary (no warning at all). What about removing the cast completely? Is there any reason to keep it?
```suggestion
					StringHelper::Sprintf("\t{ _%sSegmentRomStart, _%sSegmentRomEnd },",
```